src/HassClient/Messages/CallServiceCommand.cs

```diff
@@ -14,5 +15,8 @@ namespace JoySoftware.HomeAssistant.Messages
 
         [JsonPropertyName("service_data")]
         public object? ServiceData { get; set; }
+
+        [JsonPropertyName("target")]
+        public HassTarget? Target { get; set; }

```

Comment:
Many props in hassclient are get set. I wouls like to make them get init where possible

Reply:
Yes agree. That is a mistake from my part. We should change all to init.